.github/CONTRIBUTING.md

```diff
@@ -12,57 +12,53 @@ add a comment to it. You'll see exactly what is sent, the system is 100% transpa
+* NewPipe is translated via [Weblate](https://hosted.weblate.org/projects/newpipe/strings/). You can log in there
 with your GitHub account.
+* If the language you want to translate is not on Weblate, you can add it: see [How to add a new language](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-add-a-new-language-to-NewPipe) in the wiki.
 
 ## Code contribution
 
-* Stick to NewPipe's style conventions (well, just look the other code and then do it the same way :))
-* Do not bring non-free software (e.g., binary blobs) into the project. Also, make sure you do not introduce Google
+* Stick to NewPipe's style conventions: follow [checkStyle](https://github.com/checkstyle/checkstyle). It will run each time you build the project.
+* Do not bring non-free software (e.g. binary blobs) into the project. Also, make sure you do not introduce Google
   libraries.
 * Stick to [F-Droid contribution guidelines](https://f-droid.org/wiki/page/Inclusion_Policy)
-* Make changes on a separate branch, not on the master branch. This is commonly known as *feature branch workflow*. You
-  may then send your changes as a pull request on GitHub. Patches to the email address mentioned in this document might
-  not be considered, GitHub is the primary platform. (This only affects you if you are a member of TeamNewPipe)
+* Make changes on a separate branch with meaningful name, not on the master neither dev branch. This is commonly known as *feature branch workflow*. You
+  may then send your changes as a pull request (PR) on GitHub.
 * When submitting changes, you confirm that your code is licensed under the terms of the
   [GNU General Public License v3](https://www.gnu.org/licenses/gpl-3.0.html).
 * Please test (compile and run) your code before you submit changes! Ideally, provide test feedback in the PR
   description. Untested code will **not** be merged!
 * Try to figure out yourself why builds on our CI fail.
 * Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job,
-  but if not, you are asked to merge the master branch manually and resolve the problems on your own. That will make the
+  but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That will make the
   maintainers' jobs way easier.
 * Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for
   the core developers, and just adds work. If you do not intend to maintain features you contributed, please think again
   about submission, or clearly state that in the description of your PR.
 * Respond yourselves if someone requests changes or otherwise raises issues about your PRs.
-* Check if your contributions align with the [fdroid inclusion guidelines](https://f-droid.org/en/docs/Inclusion_Policy/).
```

Comment:
We should keep this line and use F-Droid instead of fdroid

Reply:
It is already mentioned in [line 39](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#L39).

> - Stick to F-Droid contribution guidelines